ssl_league_msgs/msg/VisionGeometryCameraCalibration.msg

```diff
@@ -0,0 +1,7 @@
+uint32 camera_id
+float32 focal_length
+geometry_msgs/Point32 principal_point
+float32 distortion
+geometry_msgs/Quaternion q
```

Comment:
Thoughts on combining `q` and `t` using `geometry_msgs/msg/Pose`?
https://docs.ros2.org/galactic/api/geometry_msgs/msg/Pose.html